docker-compose.yml

```diff
@@ -3,6 +3,7 @@ version: "3.9"
 services:
   playground:
     command: bash -c "cd /usr/src/app && make run"
+    privileged: true
```

Comment:
why is this needed?

Reply:
Isolate doesn't work without privileged mode in Docker.

Reply:
Isolate throws "Cannot run proxy, clone failed: Operation not permitted".

Reply:
I don't know how the current production works, but without the privileged flag, cgroups used by the isolate doesn't work.

Reply:
I think the Docker file is not used, it is just a convenience for developing the playground locally.